src/methods/receiver.md

```diff
@@ -17,3 +17,11 @@ are other possible receivers for a method:
 Beyond variants on `self`, there are also
 [special wrapper types](https://doc.rust-lang.org/reference/special-types-and-traits.html)
 allowed to be receiver types, such as `Box<Self>`.
+
+<details>
+
+* In reality, all functions in rust are static.  Receivers just add the "dot syntax" syntactic sugar by implicitly passing the struct by move, reference, or mutable reference.
```

Comment:
That's not really true, method calls on trait objects are dynamic.

Reply:
Right, we should instead talk about the syntax, which allows you to call methods using the fully qualified syntax.